### PR TITLE
chore: upgrade python to latest version

### PR DESCRIPTION
**Pull request by Ian2012**
## Description

Create a config to be able to modify the oars docker image.

Upgrade python to 3.8.16

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @Ian2012!  Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: tutoroars/patches/k8s-jobs [hunk 1]
```diff
@@ -14,7 +14,7 @@ spec:
         env:
           - name: VENV_DIR
             value: /opt/venv
-        image: python:3.8.10
+        image: {{DOCKER_IMAGE_OARS}}
```
Comment (mariajgrimaldi) [inline tutoroars/patches/k8s-jobs:17]:
```suggestion
        image: {{ DOCKER_IMAGE_OARS }}
```

Comment (mariajgrimaldi) [reply tutoroars/patches/k8s-jobs:17]: Same with the others

File: tutoroars/plugin.py [hunk 2]
```diff
@@ -21,6 +21,7 @@
         # Each new setting is a pair: (setting_name, default_value).
         # Prefix your setting names with 'OARS_'.
         ("OARS_VERSION", __version__),
+        ("DOCKER_IMAGE_OARS", "python:3.8.16"),
```
Comment (mariajgrimaldi) [inline tutoroars/plugin.py:24]: what if we try without the patch? so we always use the latest: https://hub.docker.com/layers/library/python/3.8

**Review by bmtcril — APPROVED** [review]
LGTM

**Review by mariajgrimaldi — APPROVED** [review]

**Comment by Ian2012** [comment]
@bmtcril ready to be merged

**Comment by openedx-webhooks** [comment]
@Ian2012 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/tutor-contrib-oars&entry.867055334=https://github.com/openedx/tutor-contrib-oars/pull/41&entry.1484655318=https://github.com/Ian2012&entry.752974735=2023-05-10+20:04&entry.1917517419=2023-05-11+18:55&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**